.gitignore

```diff
@@ -17,4 +17,10 @@
 /tmp
 
 # Ignore sensitive config files
-/config/database.yml
\ No newline at end of file
+/config/database.yml
+
+# Ignore editor temp files
```

Comment:
Since these files are made by your specific editor and system, I'd recommend placing them in a [global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) in your home directory - there's no guarantee that every developer for this app will need to ignore those files.